odoo/addons/base/models/res_partner.py

```diff
@@ -147,6 +148,10 @@ def _name_search(self, name, domain=None, operator='ilike', limit=None, order=No
             domain = [('name', operator, name)] + domain
         return self._search(domain, limit=limit, order=order)
 
+    @api.model
+    def _search_display_name(self, operator, value, limit=None):
+        search_criteria = self.env['res.partner.category'].search([('name', 'ilike', value)])
```

Comment:
Still not respecting search parameters (if I search for display_name != 'dato', I am going to be surprised with the results ^^ )

Reply:
@tde-banana-odoo ahh, i actually forgot to push a new code :(